job_creator.php

```diff
@@ -67,6 +67,7 @@ public function getInstallerVersion(): string
                 'silverstripe-module',
                 'silverstripe-recipe',
                 'silverstripe-theme',
+
             ];
             if (!isset($json->type) || !in_array($json->type, $silverstripeRepoTypes)) {
                 return '';
```

Comment:
I've moved the `!in_array($this->repoName, FORCE_INSTALLER_UNLOCKEDSTEPPED_REPOS)` to here.

End result is that serve + behat-extension will still get installer dynamically added as a dep in CI

> explicit dev dependencies for the relevant repositories

But installer is not a real dev dependency.  I also don't want to touch either serve + behat-extension because they're both fine, issue is just CI